### ISSUE TITLE
MYSQL ?

### ISSUE THREAD

**Issue by matthewjcooper** [jsreport/jsreport#267]
Is there a storage provider for mysql?

**Comment by pofider** [comment]
No, there are mssql, postgres, mongodb and file system based stores currently implemented as you can see here https://jsreport.net/learn/extensions

We don't have this on close roadmap, maybe you can use different store? Or you can try to add the mysql support based on existing postgres store on your own - https://github.com/jsreport/jsreport-postgres-store

In every case we keep this issue here and try to fit it to our roadmap.

**Comment by mtompkins** [comment]
FYI - the lack of formal support of mysql and derivatives (e.g. mariadb) makes jsreport a non-starter. Oracle and forks should be an out-of-the-gate priority. It is worrying that you "don't have this on close roadmap" as well.

This isn't a complaint, just advising this decision is costing you a likely customer at the moment. 

Cheers.
